test/ng/compileSpec.js

```diff
@@ -11441,6 +11441,40 @@ describe('$compile', function() {
     });
   });
 
+  describe('ngAutobind attributes', function() {
+    it('should expand `ng-autobind-foo` to a one-way binding and an output expression', function() {
+      var testController;
+
+      module(function($compileProvider) {
+        $compileProvider.component('test', {
+          bindings: {
+            foo: '<',
+            fooChange: '&'
+          },
+          controller: function($attrs) {
+            testController = this;
```

Comment:
We usually use `element.controller('test')` for accessing the controller. It is a good idea to keep the consistency.